index.html

```diff
@@ -1,258 +1,322 @@
+          <img
+            src="image/rectangle.png"
+            alt="parfait image"
+            class="rectangle"
+          />
+          <h2 class="heading1">Language</h2>
+          <ul>
+            <li>JavaScript</li>
+            <li>Ruby</li>
+            <li>Html</li>
+            <li>Css</li>
+          </ul>
+        </div>
+        <div class="frameworks">
+          <img src="image/square.png" alt="square image" class="square" />
+          <h2>Frameworks</h2>
+          <ul>
+            <li>Bootstrap</li>
+            <li>Ruby on Rails</li>
+            <li>RSPec</li>
+            <li>CapyBara</li>
+            <li>Selenium</li>
+          </ul>
+        </div>
+        <div class="skills">
+          <img src="image/circle.png" alt="circle image" class="circle" />
+          <h2>Skills</h2>
+          <ul>
+            <li>Codekit</li>
+            <li>github</li>
+            <li>Codepen</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
+          </ul>
+        </div>
+      </div>
+    </section>
+
+    <footer>
+      <div class="footer-content">
+        <p>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </p>
+
+        <form action="https://formspree.io/f/mayvddjz" method="post">
+          <input
+            type="text"
+            placeholder="Full name"
+            name="Full name"
+            maxlength="30"
+            required
+          />
+          <input
+            type="email"
+            placeholder="Email address"
+            name="email"
+            required
+          />
+          <textarea
+            cols="30"
+            class="textarea"
+            rows="10"
+            placeholder="Enter text here"
+            maxlength="500"
+            required
+            name="Message"
+          ></textarea>
+          <button class="form-button" type="submit">Get in touch</button>
+        </form>
+      </div>
+      <img
+        class="hide-desktop"
+        src="./image/footerIcon2/Rectangle42.png"
+        alt="Rectangle 42"
+      />
+      <div class="footer-line hide-mobile"></div>
+
+      <ul class="sociallist">
+        <li>
+          <img src="./image/footerIcon2/Icon-GitHub.png" alt="Github icon" />
+        </li>
+        <li>
+          <img
+            src="./image/footerIcon2/Icon-Linkedin.png"
+            alt="Linkedin Icon"
+          />
+        </li>
+        <li>
+          <img
+            src="./image/footerIcon2/Icon-Angelist.png"
+            alt="Angelist Icon"
+          />
+        </li>
+        <li>
+          <img src="./image/footerIcon2/Icon-Twitter.png" alt="Twitter Icon" />
+        </li>
+        <li>
+          <img src="./image/footerIcon2/Icon-Medium.png" alt="Medium Icon" />
+        </li>
+      </ul>
+      <img
+        class="shape hide-desktop"
+        src="./image/footerIcon2/Shape.png"
+        alt="Shape Icon"
+      />
+    </footer>
+  </body>
```

Comment:
Mobile Version:
- [ ] Kindly make sure to place the entire recent works in the centre as per the Figma design, I have attached a video to refer to what your issues are.
- [ ] Kindly reduce the extra space after about me and place the background image in about me a bit upwards as per the Figma design.
- [ ] Kindly place all the li in language, frameworks and skills in the centre as per the Figma design.
- [ ] Kindly place the contact section in the centre as per the Figma design and also make sure to align the text field and make it equal in width with the name and email fields.
- [ ] Kindly place the line above social media icons in centre as per the Figma design.

Your Mobile version:

https://user-images.githubusercontent.com/70269507/161405973-f02429e8-58cd-44ec-82f5-a17e9f57db27.mov

Desktop Version:
- [ ] Kindly make sure to fix the entire section of recent works overlapping as shown in the video and make it displayed in the centre.
- [ ] Kindly fix the languages section and contact section and place them in the centre. As you can see it's not displayed in a small size.
- [ ] Kindly make sure to reduce the extra spacing above the language section and below the about section.
- [ ] Kindly add the line before the social media icons as per the Figma design.

Your Desktop version-

https://user-images.githubusercontent.com/70269507/161405981-bbfbc696-e84c-4746-912e-02cde3a696a5.mov

## You can refer [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) for clarifications.